### PR TITLE
started using absolute dates for summary

### PR DESCRIPTION
**Pull request by smitthakkar96**

### CODE REVIEW

**Comment by nigelbabu** [comment]
I don't understand what you're trying to do with this change.

**Comment by smitthakkar96** [comment]
the issue asks to use absolute dates am I right? So what I did was I tried to get the last_monday and then I subtracted the num weeks, this way we are avoiding relative dates. Isn't that what you expected?

File: fstat/lib.py [hunk 1]
```diff
@@ -7,3 +7,13 @@ def x_days_ago(x=1):
     '''
     return (datetime.today().replace(hour=0, minute=0, second=0,
             microsecond=0) - timedelta(days=x))
+
+
+def x_weeks_ago(x=1):
+    '''
+        Return timestamp for X weeks ago
```
Comment (nigelbabu) [inline fstat/lib.py]: You might want to say Monday X weeks ago

File: fstat/controller.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ def index():
 def weekly_overall_summary(num=None):
     if num > 12:
         num = 12
-    cut_off_date = (x_days_ago(num*7))
+    cut_off_date = (x_weeks_ago(num*7))
```
Comment (nigelbabu) [inline fstat/controller.py]: This is why I got confused. You probably don't need the `(num*7)`

File: fstat/lib.py [hunk 3]
```diff
@@ -7,3 +7,13 @@ def x_days_ago(x=1):
...
+    '''
+    today = datetime.today().replace(hour=0, minute=0, second=0,
+                                     microsecond=0)
+    last_monday = today - timedelta(days=today.weekday())
```
Comment (nigelbabu) [inline fstat/lib.py]: Will this show failures from today after midnight?

**Pull request closed without merging**